### PR TITLE
Update pipeline images to the 0.2.0 release.

### PR DESCRIPTION
**Pull request by jlewi**
related to kubeflow/manifests#796

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/manifests/802)

### CODE REVIEW

**Comment by Bobgy** [comment]
Thanks @jlewi!

Besides the changes you made, I think you also need to add the following two commits.

This commit adds pod delete permission to pipeline api service. The permission is used for workflow retry feature.
https://github.com/Bobgy/manifests/pull/2/commits/c27bb576b0b4fbeabadcfc9ac0e53570d564094b

This commit lets argo workflow permission get collected by cluster-admin, cluster-editor and cluster-viewer. (Not directly related to KFP, but I think it's useful.)
https://github.com/Bobgy/manifests/pull/2/commits/9ef7687384479a6fc77526f5eaf0e6d1d25368f8

(I've been experimenting KFP multi user based on Kubeflow 0.7. These are the only fixes I needed to run KFP 0.1.40 on Kubeflow 0.7. Other changes are only related to multi user.)

**Comment by jlewi** [comment]
Test failure looks like kubeflow/manifests#806

**Comment by jlewi** [comment]
Thanks @Bobgy  PTAL

File: pipeline/api-service/base/role.yaml [hunk 1]
```diff
@@ -1,4 +1,6 @@
 apiVersion: rbac.authorization.k8s.io/v1beta1
+# TODO: Does this need to be changed to a clusterrole?
+# see  manifests in kubeflow/pipelines
```
Comment (Bobgy) [inline pipeline/api-service/base/role.yaml]: No need so far. Clusterrole needed when multi user mode enabled.

**Pull request merged**